server/graphql/v2/object/Individual.js

```diff
@@ -107,6 +108,12 @@ export const Individual = new GraphQLObjectType({
           }
         },
       },
+      hasSeenLatestChangelogEntry: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        async resolve(individual) {
```

Comment:
In other resolvers we name it `collective` so it's clear it's a `Collective`.

Reply:
Done. 😄